tests/test_more.py

```diff
@@ -4137,7 +4137,7 @@ def test_samling_entire_iterable(self):
         self.assertEqual(actual, expected)
 
     def test_scale_invariance_of_weights(self):
-        """The probabilit of chosing element a_i is w_i / sum(weights).
+        """The probabilit of choosing element a_i is w_i / sum(weights).
```

Comment:
It didn't complain about "probabilit"?

Reply:
It is based on a dictionary of typos. This typo is not in the dictionary. Will fix it.